collisionmanager.cpp

```diff
@@ -9,7 +9,7 @@
 namespace pong
 {
 
-void CollisionManager::ProcessCollisions(const std::vector<std::unique_ptr<GameObject>>& gameObjects)
+void CollisionManager::ProcessCollisions(const std::vector<std::shared_ptr<GameObject>>& gameObjects)
```

Comment:
Can you justify your switching to shared ptr for me? Why does it make sense for these objects to have shared ownership instead of single ownership?

(not critiquing you, just testing your knowledge and seeing what thought you've put into it)

Reply:
Check out my one of my comments I put here! I was curious of your thoughts on this very thing.

Reply:
I am missing the comment, I think, could you link me or give me a line number?

Reply:
Yeah I left a few, weid that it is not showing up.  Here is that question https://github.com/jonmgomez/pong/pull/7#discussion_r1385685102

and the other comments
https://github.com/jonmgomez/pong/pull/7#discussion_r1385685057
https://github.com/jonmgomez/pong/pull/7#discussion_r1385685128
https://github.com/jonmgomez/pong/pull/7#discussion_r1385685159
let me know if it still gives problems, I can recomment them if need be

Reply:
Got them now!! Thanks for the links